tests/unit_tests/CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ set(rng_TEST_LIST
 
 set(rng_TEST_LINK "")
 
+# DFT config
+set(dft_TEST_LIST
+      dft_source)
+
+set(dft_TEST_LINK "")
+
```

Comment:
I think all this block is unused left over.

Reply:
Ok, `dft_TEST_LINK` is set twice though.

Reply:
In these 6 lines one variable is `LIST` and the other is `LINK`.

Reply:
My bad I was confused with the `rng` one above!